### PR TITLE
Fix Travis

### PR DESCRIPTION
**Pull request by Hadis-Fard**
- Run server image before building client

- Added sleep to before_script in travis to let the server fire up

- Added entrypoint script to test connection at client container run time

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,22 +15,27 @@ env:
...
 script: 
-  - docker run -e TRAVIS_JOB_ID -t -d -w $PHPSQLDIR --link $TEST_PHP_SQL_SERVER --name=client msphpsql-dev
-  - docker ps -a 
+  - docker run -e TRAVIS_JOB_ID -t -d -w $PHPSQLDIR --name=client --link $TEST_PHP_SQL_SERVER msphpsql-dev
+  - docker ps -a
+  - docker logs client
   - docker exec client python ./test/functional/setup/setup_dbs.py -dbname $SQLSRV_DBNAME
   - docker exec client python ./test/functional/setup/setup_dbs.py -dbname $PDOSQLSRV_DBNAME
-  - docker exec client php ./source/pdo_sqlsrv/run-tests.php ./test/functional/pdo_sqlsrv/*.phpt
+  - docker exec client php ./source/sqlsrv/run-tests.php ./test/functional/pdo_sqlsrv/*.phpt
```
Comment (david-puglielli) [inline .travis.yml]: This uses the same run-tests.php for sqlsrv and pdo_sqlsrv tests?

File: entrypoint.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+set -e
+
+testConnection="/opt/mssql-tools/bin/sqlcmd -S sql -U sa -P Password123"
+
+until $testConnection; do
+>&2 echo "SQL Server is starting up.."
+sleep 1
+done
```
Comment (david-puglielli) [inline entrypoint.sh]: Is there a chance this could result in an infinite loop if there is some other remote server problem?

File: .travis.yml [hunk 3]
```diff
@@ -15,22 +15,27 @@ env:
   - PHPSQLDIR=/REPO/msphpsql-dev
   - TEST_PHP_SQL_SERVER=sql
   - SQLSRV_DBNAME=msphpsql_sqlsrv
-  - PDOSQLSRV_DBNAME=msphpsql_pdosqlsrv  
+  - PDOSQLSRV_DBNAME=msphpsql_pdosqlsrv
```
Comment (lilgreenbird) [inline .travis.yml]: what changed here??

File: entrypoint.sh (hunk 2 above)
Comment (lilgreenbird) [inline entrypoint.sh]: this seems to be a very tight loop and as David suggested may result in an infinite loop if the server startup failed for whatever reason. I would set a limit and increase the wait.
Actually there's a travis_retry can we make use of that?

File: .travis.yml (hunk 3 above)
Comment (Hadis-Fard) [inline .travis.yml]: space at the end

File: .travis.yml (hunk 1 above)
Comment (Hadis-Fard) [inline .travis.yml]: good catch

File: entrypoint.sh (hunk 2 above)
Comment (Hadis-Fard) [inline entrypoint.sh]: updated please review! travis_retry is used when you install dependencies within travis and it gives timeout. in this case we install dependencies within docker

**Comment by Hadis-Fard** [comment]
@david-puglielli @lilgreenbird Could you review please? would like to push this before other PRs?

**Pull request merged**